core-java-modules/core-java-graphql/src/main/java/com/baeldung/graphql/clients/AmericanExpressNodes.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.graphql.clients;
+
+import com.baeldung.graphql.Constants;
+import com.baeldung.graphql.data.Data;
+import io.aexp.nodes.graphql.GraphQLRequestEntity;
+import io.aexp.nodes.graphql.GraphQLResponseEntity;
+import io.aexp.nodes.graphql.GraphQLTemplate;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+
+public final class AmericanExpressNodes {
```

Comment:
Let's ditch the `final` and `private` constructor here too - they're unnecessary noise.